src/cli.rs

```diff
@@ -46,6 +46,7 @@ pub fn build_cli() -> App<'static, 'static> {
                         .short("o")
                         .long("output-dir")
                         .takes_value(true)
+                        .default_value("$PROJECT_ROOT/public")
```

Comment:
Or may be it should be `project_root/public` as `$PROJECT_ROOT` may lead to a confusion of `PROJECT_ROOT` being an acceptable variable by zola?